src/lib/ng-package/entry-point/write-package.transform.ts

```diff
@@ -128,6 +74,70 @@ export const writePackageTransform = (options: NgPackagrOptions) =>
     return graph;
   });
 
+async function copyAssets(
+  graph: BuildGraph,
+  entryPointNode: EntryPointNode,
+  ngPackageNode: PackageNode,
+): Promise<void> {
+  const ngPackage = ngPackageNode.data;
+
+  const pathsIgnored: string[] = [
+    '.gitkeep',
+    '**/.DS_Store',
+    '**/Thumbs.db',
+    '**/node_modules/**',
+    `${ngPackage.dest}/**`,
+  ];
+
+  const assets: AssetEntry[] = [];
+
+  for (const item of ngPackage.assets.concat('LICENSE', '**/README.md')) {
+    let asset: AssetEntry;
+    if (typeof item == 'object') {
+      asset = item;
+      asset.glob = path.join(ngPackage.src, asset.input, asset.glob);
```

Comment:
This seems to be redundant. 

```suggestion
```

Reply:
Without this statement, the result paths of the glob will be relative rather than absolute.

Reply:
Without this statement, the result paths of the glob will be relative rather than absolute.